osu.Game/Screens/Edit/Editor.cs

```diff
@@ -721,10 +721,16 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
             switch (e.Action)
             {
                 case GlobalAction.EditorSeekToPreviousHitObject:
+                    if (editorBeatmap.SelectedHitObjects.Any())
+                        return false;
```

Comment:
I don't see how more customisability amounts to a 'fail'. More customisability is often better.
I guess most of your issue lies with the implementation of my suggestion as it would probably be a pain.

Reply:
> To me a makes a lot more sense if all the seek keybinds adhere to the same rules:
> - No conflicted keybind -> works in all contexts
> - Conflicted keybind -> only works if there is no selection

No I do not accept this premise. Doing this is a failure of design. 

There are two options: 
- You make things customisable but also make sure to put enough guardrails to ensure the user can't shoot their foot off with the customisation options. 
- You don't make things customisable at all. 

I'm not having this halfway "you make things customisable but by doing so you allow other features to silently change behaviour for no observable reason" you're proposing here. If you insist you can try implementing it yourself but I will not put my hand to it by either trying to implement this or approving it during review.